OpenRA.Mods.Common/Traits/Modifiers/FrozenUnderFog.cs

```diff
@@ -64,13 +63,47 @@ public FrozenUnderFog(ActorInitializer init, FrozenUnderFogInfo info)
 			footprint = footprintCells.SelectMany(c => map.ProjectedCellsCovering(c.ToMPos(map))).ToArray();
 		}
 
+		public void Created(Actor self)
```

Comment:
Shouldn't be a problem in this case, just making a note for anyone who might be following the notification chain.